src/components/Navbar.tsx

```diff
@@ -14,18 +16,18 @@ const Navbar = () => {
 
   return (
     <nav
-      className="top-0 z-10 flex h-[60px] w-full flex-row items-center 
-                    justify-between border border-dark-tan bg-med-tan"
+      className="\ top-0 z-10 flex h-[60px] w-full flex-row items-center 
+                    justify-between border border-dark-tan bg-med-tan px-24 py-2 pt-3"
     >
-      <div className="pl-[20px] font-serif text-xl font-medium sm:pl-[40px] md:text-2xl">
-        <Link href="/">The Legacy Project</Link>
-      </div>
+      <Link href="/public" onClick={() => setDropdownVisible(false)}>
+        <Image src={logoicon} alt="Legacy Logo" className="w-[200px]" />
+      </Link>
 
       <div className="visible z-10 pr-[20px] sm:pr-[40px]">
```

Comment:
`pr-[20px] sm:pr-[40px]` is already here, but I think the extra padding looks a bit weird. What do you guys think about removing it?